src/docs/blueprint_download.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Eclipse Theia Blueprint Product
+---
+
+# Eclipse Theia Blueprint Product
+
+<img src="/theia-logo-blueprint.png" alt="Theia Logo Blue" style="max-width: 525px">
+
+## What is this?
+
+Eclipse Theia Blueprint product is a **template** to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform. Theia Blueprint assembles a selected subset of existing Theia features and extensions. We provide installers for Theia Blueprint to be downloaded (see below) as well as documentation on how to customize this template to build a product and installers for your own Theia-based product.
+
+## What is it not?
+
+Eclipse Theia Blueprint product is ***not*** **a production-ready product**. Therefore, it is also not a replacement for Visual Studio Code or any other IDE.
+
+## Downloads
+
+<span style="color:red">Warning: Currently Eclipse Theia Blueprint product is an **alpha** version. Please consider the current versions as experimental. </span>
+
+<table cellspacing="25">
+  <thead>
+    <tr>
+      <th>Windows</th>
+      <th>Linux</th>
+      <th>MacOS</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.0/windows/Theia-1.9.0.exe&r=1">Download 1.9.0</a></td>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.0/linux/Theia-1.9.0.AppImage&r=1">Download 1.9.0</a></td>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.0/macos/Theia-1.9.0.pkg&r=1">Download 1.9.0</a></td>
+    </tr>
+  </tbody>
+</table>
+
+## Reporting feature requests and bugs
+
+The features in the Eclipse Theia Blueprint product are based on Theia and the included extensions/plugins. For bugs in Theia please consider opening an issue in the [Theia project on Github](https://github.com/eclipse-theia/theia/issues/new/choose). 
```

Comment:
I believe the goal is not to have users report `blueprint` problems directly in the upstream repository, but `blueprint` itself (given that it is also not a production-ready product). Can you confirm @marcdumais-work.

Reply:
I think `blueprint`-specific issues should be opened in that repo. But here the mention is for `bugs in Theia`, which indeed are better opened in the main Theia repo. I am not sure how useful it is to mention issues here. Maybe it could be rephrased more generally, still pointing to the Theia repo? e.g. "For more details please see the [Eclipse Theia  repo](https://github.com/eclipse-theia/theia/)"